salt/orchestrate/aws/rds.sls

```diff
@@ -67,16 +67,23 @@ create_{{ ENVIRONMENT }}_{{ dbconfig.name }}_rds_store:
     - require:
         - boto_rds: create_{{ ENVIRONMENT }}_rds_db_subnet_group
 
+{% set mount_point = '{}-{}-{}'.format(dbconfig.engine, ENVIRONMENT, dbconfig.name) %}
 configure_vault_postgresql_{{ dbconfig.name }}_backend:
   vault.secret_backend_enabled:
-    - backend_type: postgresql
-    - description: Backend to create dynamic PostGreSQL credentials for {{ ENVIRONMENT }}
-    - mount_point: postgresql-{{ ENVIRONMENT }}-{{ dbconfig.name }}
+    - backend_type: database
+    - description: Backend to create dynamic {{ dbconfig.engine }} credentials for {{ ENVIRONMENT }}
+    - mount_point: {{ mount_point }}
     - ttl_max: {{ SIX_MONTHS }}
     - ttl_default: {{ SIX_MONTHS }}
     - lease_max: {{ SIX_MONTHS }}
     - lease_default: {{ SIX_MONTHS }}
+    - connection_config_path: 
```

Comment:
What does this serve?

Reply:
That is me forgetting to add the parameter that gets passed to it... I'll fix that.